example-arcgis-js-api/bower.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "demo-app",
+  "version": "0.0.1",
```

Comment:
version is depricated: https://github.com/bower/spec/blob/master/json.md#version